src/main/java/net/minecraftforge/oredict/OreDictionary.java

```diff
@@ -77,43 +81,57 @@ public static void initVanillaEntries()
             registerOre("oreEmerald",  Blocks.emerald_ore);
             registerOre("oreQuartz",   Blocks.quartz_ore);
             registerOre("oreCoal",     Blocks.coal_ore);
-            registerOre("blockGold",     Blocks.gold_block);
-            registerOre("blockIron",     Blocks.iron_block);
-            registerOre("blockLapis",    Blocks.lapis_block);
-            registerOre("blockDiamond",  Blocks.diamond_block);
-            registerOre("blockRedstone", Blocks.redstone_block);
-            registerOre("blockEmerald",  Blocks.emerald_block);
-            registerOre("blockQuartz",   Blocks.quartz_block);
-            registerOre("blockCoal",     Blocks.coal_block);
-            registerOre("blockGlassColorless", Blocks.glass);
-            registerOre("blockGlass",    Blocks.glass);
-            registerOre("blockGlass",    new ItemStack(Blocks.stained_glass, 1, WILDCARD_VALUE));
-            //blockGlass{Color} is added below with dyes
-            registerOre("paneGlassColorless", Blocks.glass_pane);
-            registerOre("paneGlass",     Blocks.glass_pane);
-            registerOre("paneGlass",     new ItemStack(Blocks.stained_glass_pane, 1, WILDCARD_VALUE));
-            //paneGlass{Color} is added below with dyes
+
+            // ingots/nuggets
             registerOre("ingotIron",     Items.iron_ingot);
             registerOre("ingotGold",     Items.gold_ingot);
             registerOre("ingotBrick",    Items.brick);
             registerOre("ingotBrickNether", Items.netherbrick);
             registerOre("nuggetGold",  Items.gold_nugget);
+
+            // gems and dusts
             registerOre("gemDiamond",  Items.diamond);
             registerOre("gemEmerald",  Items.emerald);
             registerOre("gemQuartz",   Items.quartz);
+            registerOre("gemPrismarine", Items.prismarine_shard);
+            registerOre("dustPrismarine", Items.prismarine_crystals);
             registerOre("dustRedstone",  Items.redstone);
             registerOre("dustGlowstone", Items.glowstone_dust);
             registerOre("gemLapis",    new ItemStack(Items.dye, 1, 4));
-            registerOre("slimeball",   Items.slime_ball);
-            registerOre("glowstone",   Blocks.glowstone);
+
+            // storage blocks
```

Comment:
That's what I've always heard iron/gold/diamond/whatever blocks called. Each block represents 9 ingots worth, which makes for convenient storage.

If there's a more generally accepted name for that kind of block I don't mind updating